[2021-12-16 11:17:38,944] {processor.py:163} INFO - Started process (PID=67) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:17:38,945] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:17:38,946] {logging_mixin.py:109} INFO - [2021-12-16 11:17:38,946] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:17:38,970] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:17:39,951] {logging_mixin.py:109} INFO - [2021-12-16 11:17:39,951] {providers_manager.py:156} WARNING - Exception when importing 'airflow.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-airflow-providers-google' package: No module named 'plyvel'
[2021-12-16 11:17:40,381] {logging_mixin.py:109} INFO - [2021-12-16 11:17:40,380] {providers_manager.py:156} WARNING - Exception when importing 'airflow.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-airflow-providers-google' package: No module named 'plyvel'
[2021-12-16 11:17:40,402] {logging_mixin.py:109} INFO - [2021-12-16 11:17:40,402] {manager.py:496} INFO - Created Permission View: can read on DAG:test_utils
[2021-12-16 11:17:40,412] {logging_mixin.py:109} INFO - [2021-12-16 11:17:40,412] {manager.py:496} INFO - Created Permission View: can edit on DAG:test_utils
[2021-12-16 11:17:40,414] {logging_mixin.py:109} INFO - [2021-12-16 11:17:40,413] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:17:40,423] {logging_mixin.py:109} INFO - [2021-12-16 11:17:40,423] {dag.py:2415} INFO - Creating ORM DAG for test_utils
[2021-12-16 11:17:40,434] {logging_mixin.py:109} INFO - [2021-12-16 11:17:40,434] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:17:40,449] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 1.508 seconds
[2021-12-16 11:18:11,123] {processor.py:163} INFO - Started process (PID=373) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:18:11,125] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:18:11,127] {logging_mixin.py:109} INFO - [2021-12-16 11:18:11,127] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:18:11,157] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:18:11,169] {logging_mixin.py:109} INFO - [2021-12-16 11:18:11,169] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:18:11,187] {logging_mixin.py:109} INFO - [2021-12-16 11:18:11,187] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:18:11,200] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.080 seconds
[2021-12-16 11:18:41,694] {processor.py:163} INFO - Started process (PID=433) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:18:41,695] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:18:41,697] {logging_mixin.py:109} INFO - [2021-12-16 11:18:41,697] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:18:41,725] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:18:41,735] {logging_mixin.py:109} INFO - [2021-12-16 11:18:41,735] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:18:41,753] {logging_mixin.py:109} INFO - [2021-12-16 11:18:41,753] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:18:41,762] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.072 seconds
[2021-12-16 11:19:11,835] {processor.py:163} INFO - Started process (PID=503) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:19:11,837] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:19:11,838] {logging_mixin.py:109} INFO - [2021-12-16 11:19:11,838] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:19:11,867] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:19:11,877] {logging_mixin.py:109} INFO - [2021-12-16 11:19:11,877] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:19:11,892] {logging_mixin.py:109} INFO - [2021-12-16 11:19:11,892] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:19:11,900] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.069 seconds
[2021-12-16 11:19:42,071] {processor.py:163} INFO - Started process (PID=563) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:19:42,073] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:19:42,074] {logging_mixin.py:109} INFO - [2021-12-16 11:19:42,074] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:19:42,101] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:19:42,111] {logging_mixin.py:109} INFO - [2021-12-16 11:19:42,111] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:19:42,127] {logging_mixin.py:109} INFO - [2021-12-16 11:19:42,127] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:19:42,135] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.070 seconds
[2021-12-16 11:20:12,425] {processor.py:163} INFO - Started process (PID=632) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:20:12,427] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:20:12,428] {logging_mixin.py:109} INFO - [2021-12-16 11:20:12,428] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:20:12,458] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:20:12,468] {logging_mixin.py:109} INFO - [2021-12-16 11:20:12,468] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:20:12,487] {logging_mixin.py:109} INFO - [2021-12-16 11:20:12,487] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:20:12,497] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.076 seconds
[2021-12-16 11:20:42,816] {processor.py:163} INFO - Started process (PID=692) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:20:42,818] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:20:42,820] {logging_mixin.py:109} INFO - [2021-12-16 11:20:42,819] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:20:42,848] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:20:42,859] {logging_mixin.py:109} INFO - [2021-12-16 11:20:42,858] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:20:42,874] {logging_mixin.py:109} INFO - [2021-12-16 11:20:42,874] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:20:42,882] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.070 seconds
[2021-12-16 11:21:13,271] {processor.py:163} INFO - Started process (PID=761) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:21:13,273] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:21:13,274] {logging_mixin.py:109} INFO - [2021-12-16 11:21:13,274] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:21:13,304] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:21:13,318] {logging_mixin.py:109} INFO - [2021-12-16 11:21:13,318] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:21:13,335] {logging_mixin.py:109} INFO - [2021-12-16 11:21:13,335] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:21:13,343] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.076 seconds
[2021-12-16 11:21:43,584] {processor.py:163} INFO - Started process (PID=813) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:21:43,586] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:21:43,587] {logging_mixin.py:109} INFO - [2021-12-16 11:21:43,587] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:21:43,611] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:21:43,619] {logging_mixin.py:109} INFO - [2021-12-16 11:21:43,619] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:21:43,650] {logging_mixin.py:109} INFO - [2021-12-16 11:21:43,650] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:21:43,671] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.089 seconds
[2021-12-16 11:22:13,935] {processor.py:163} INFO - Started process (PID=881) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:22:13,937] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:22:13,938] {logging_mixin.py:109} INFO - [2021-12-16 11:22:13,938] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:22:13,967] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:22:13,978] {logging_mixin.py:109} INFO - [2021-12-16 11:22:13,978] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:22:13,995] {logging_mixin.py:109} INFO - [2021-12-16 11:22:13,995] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:22:14,005] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.074 seconds
[2021-12-16 11:22:44,398] {processor.py:163} INFO - Started process (PID=952) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:22:44,400] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:22:44,401] {logging_mixin.py:109} INFO - [2021-12-16 11:22:44,401] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:22:44,426] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:22:44,436] {logging_mixin.py:109} INFO - [2021-12-16 11:22:44,436] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:22:44,459] {logging_mixin.py:109} INFO - [2021-12-16 11:22:44,459] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:22:44,469] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.074 seconds
[2021-12-16 11:23:14,626] {processor.py:163} INFO - Started process (PID=1012) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:23:14,629] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:23:14,630] {logging_mixin.py:109} INFO - [2021-12-16 11:23:14,630] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:23:14,660] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:23:14,671] {logging_mixin.py:109} INFO - [2021-12-16 11:23:14,671] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:23:14,690] {logging_mixin.py:109} INFO - [2021-12-16 11:23:14,690] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:23:14,700] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.080 seconds
[2021-12-16 11:23:45,019] {processor.py:163} INFO - Started process (PID=1081) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:23:45,022] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:23:45,023] {logging_mixin.py:109} INFO - [2021-12-16 11:23:45,023] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:23:45,054] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:23:45,066] {logging_mixin.py:109} INFO - [2021-12-16 11:23:45,066] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:23:45,089] {logging_mixin.py:109} INFO - [2021-12-16 11:23:45,089] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:23:45,099] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.084 seconds
[2021-12-16 11:24:15,614] {processor.py:163} INFO - Started process (PID=1141) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:24:15,617] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:24:15,618] {logging_mixin.py:109} INFO - [2021-12-16 11:24:15,618] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:24:15,648] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:24:15,658] {logging_mixin.py:109} INFO - [2021-12-16 11:24:15,658] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:24:15,675] {logging_mixin.py:109} INFO - [2021-12-16 11:24:15,675] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:24:15,685] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.075 seconds
[2021-12-16 11:24:46,425] {processor.py:163} INFO - Started process (PID=1211) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:24:46,427] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:24:46,429] {logging_mixin.py:109} INFO - [2021-12-16 11:24:46,428] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:24:46,455] {processor.py:637} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:24:46,463] {logging_mixin.py:109} INFO - [2021-12-16 11:24:46,463] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-16 11:24:46,476] {logging_mixin.py:109} INFO - [2021-12-16 11:24:46,476] {dag.py:2928} INFO - Setting next_dagrun for test_utils to None
[2021-12-16 11:24:46,484] {processor.py:171} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py took 0.062 seconds
[2021-12-16 11:25:16,979] {processor.py:163} INFO - Started process (PID=1272) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:25:16,981] {processor.py:625} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-16 11:25:16,983] {logging_mixin.py:109} INFO - [2021-12-16 11:25:16,983] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/test_utils.py
[2021-12-16 11:25:17,005] {processor.py:174} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2336, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 508, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres" to address: Name or service not known


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/dag_processing/processor.py", line 168, in _run_file_processor
    callback_requests=callback_requests,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/dag_processing/processor.py", line 634, in process_file
    self._deactivate_missing_dags(session, dagbag, file_path)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/dag_processing/processor.py", line 670, in _deactivate_missing_dags
    .update({DagModel.is_active: False}, synchronize_session="fetch")
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 4063, in update
    update_op.exec_()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1696, in exec_
    self._do_pre_synchronize()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1806, in _do_pre_synchronize
    select_stmt, mapper=self.mapper, params=query._params
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1295, in execute
    return self._connection_for_bind(bind, close_with_result=True).execute(
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1151, in _connection_for_bind
    engine, execution_options
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 433, in _connection_for_bind
    conn = bind._contextual_connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2302, in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2340, in _wrap_pool_connect
    e, dialect, self
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1584, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2336, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 508, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Name or service not known

(Background on this error at: http://sqlalche.me/e/13/e3q8)
